Makefile

```diff
@@ -11,11 +11,13 @@ GCC = g++
 endif
 LD = $(GCC)
 
-ifndef RELEASE
-CFLAGS += -Werror -Wimplicit-fallthrough
+ifdef LLD
 LDFLAGS += -fuse-ld=lld
 endif
 
+ifndef RELEASE
+CFLAGS += -Werror -Wimplicit-fallthrough -Wno-unused-private-field -Wno-unused-function
```

Comment:
Can you remove -Wno-unused-private-field? I removed the fields that were causing the error.

Reply:
Sure.